src/app/test-results/test-results-list/test-results-detail-list.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ResponseTestResultDetail, TestResultService } from 'api-client';
+
+@Component({
+  selector: 'wh-test-results-detail-list',
+  templateUrl: './test-results-detail-list.component.html',
+})
+export class TestResultsDetailListComponent implements OnInit {
+  @Input() buildId: number;
+  @Input() summaryId: number;
+  details: ResponseTestResultDetail[];
+  failedDetails: ResponseTestResultDetail[];
+  private fetched: boolean;
```

Comment:
nit: Suggest renaming this field `fetched`&rarr;`hasFetched`. Booleans are conventionally prefixed with `has` or `is` to increases readability. Go is like the only language that does not have this convention.

It's kind of like hungarian notation, but only for bools.

Maybe this is just my C# background talking though, as in C# the `is` and `has` prefixes are borderline rules instead of conventions.